src/set_theory/zfc.lean

```diff
@@ -682,19 +682,27 @@ def Class := set Set
 
 namespace Class
 
+/-- The preferred way to convert a set of Sets into a Class. -/
+def of_set : set Set ≃ Class := equiv.refl _
+
+/-- The preferred way to state that a set `x` belongs in a class `A`. -/
+def mem_set (x : Set.{u}) (A : Class.{u}) : Prop := A x
+
+@[simp] theorem apply_iff_mem_set {x : Set.{u}} {A : Class.{u}} : A x ↔ mem_set x A := iff.rfl
+
+@[simp] theorem mem_set_of_set {x : Set} {s : set Set} : mem_set x (of_set s) ↔ x ∈ s := iff.rfl
+
 /-- Coerce a ZFC set into a class -/
-def of_Set (x : Set.{u}) : Class.{u} := {y | y ∈ x}
-instance : has_coe Set Class := ⟨of_Set⟩
+instance : has_coe Set Class := ⟨λ x, {y | y ∈ x}⟩
 
 /-- The universal class -/
-def univ : Class := set.univ
+def univ : Class := of_set set.univ
 
 /-- Assert that `A` is a ZFC set satisfying `p` -/
-def to_Set (p : Set.{u} → Prop) (A : Class.{u}) : Prop := ∃ x, ↑x = A ∧ p x
+def to_Set (p : Class.{u}) (A : Class.{u}) : Prop := ∃ x, ↑x = A ∧ mem_set x p
```

Comment:
Note that this is no longer a very useful definition. If this PR is merged, it will be ditched in favor of `∈` in a followup PR.